app-misc/ca-certificates/files/certdata2pem.py

```diff
@@ -115,9 +115,8 @@
                                       .replace('(', '=')\
                                       .replace(')', '=')\
                                       .replace(',', '_') + '.pem'
-        fname = fname.decode('string_escape')
+        fname = fname.encode('utf8').decode('unicode-escape')
         f = open(fname, 'w')
         f.write("-----BEGIN CERTIFICATE-----\n")
-        f.write("\n".join(textwrap.wrap(base64.b64encode(obj['CKA_VALUE']), 64)))
+        f.write("\n".join(textwrap.wrap(base64.b64encode(obj['CKA_VALUE'].encode('latin1')).decode('utf8'), 64)))
```

Comment:
I rewrote this to make more sense out of it.

Reply:
I think the new version makes more sense, thanks!

Reply:
FWIW: i found another transform that would have worked. Since the previous version used `value += chr(0-255)`, `bytes(map(ord, value))` returns the numbers.

Reply:
The comment helps, thanks

Reply:
> FWIW: i found another transform that would have worked. Since the previous version used `value += chr(0-255)`, `bytes(map(ord, value))` returns the numbers.

I'd say storing numbers in a string (as done through `chr`) only to transform that back to into bytes was a mistake. The numbers should be treated as bytes from the beginning.